cifsd/rpc_srvsvc.c

```diff
@@ -280,7 +280,7 @@ static int srvsvc_parse_share_info_req(struct cifsd_dcerpc *dce,
 		ptr = ndr_read_int32(dce); // read container array pointer
 					   // it should be null
 		if (ptr != 0x00) {
-			pr_err("SRVSVC: container array pointer is %p\n",
+			pr_err("SRVSVC: container array pointer is %x\n",
```

Comment:
What kind of problem does this fix?

-ss

Reply:
wrong format. ptr is an unsigned int. GCC and Clang warn with -Wformat

Reply:
OK.

Wformat warnings are from "extra warnings" category. Well, because of cases like this.
We can't have negative `ptr`, so "**signed int** to **unsigned long**" promition is not
really possible, and, in fact, we care only about "!= 0" case. Sometimes warnings are
just "noise". But OK, can take it in.

Please copy-patse gcc/clang warnings into the commit message next time.

-ss